lib/sensors/mux.cpp

```diff
@@ -10,14 +10,15 @@ Mux<T, N>::Mux(io::I2c &i2c,
       i2c_(i2c),
       sensors_(std::move(sensors))
 {
+  static_assert(N <= 8, "Mux can only have up to 8 channels");
```

Comment:
But `static_cast<std::uint8_t>(1 << 8) == 0`?

Reply:
whoops, that equals to was copilot

Reply:
but you can't have more sensors than you have channels